app/graphql/mutations/profile/create.rb

```diff
@@ -18,26 +18,36 @@ class Create < BaseMutation
       field :profile, Types::Profile, null: false
 
       def resolve(args = {})
-        profile = ::Profile.new(new_profile_options(args))
-        profile.save!
-        profile.update_rules(ref_ids: args[:selected_rule_ref_ids])
+        policy = ::Policy.new(new_policy_options(args))
+        profile = ::Profile.new(new_profile_options(args)).fill_from_parent
+
+        Policy.transaction do
+          policy.save!
+          profile.update!(policy: policy)
+          profile.update_rules(ref_ids: args[:selected_rule_ref_ids])
+        end
+
         { profile: profile }
```

Comment:
Note that policy information (i.e. business_objective, compliance_threshold, etc.) is communicated via the profile, even though those columns still exist on profiles (for now).